### ISSUE TITLE
Failed to perform code-generation error needs improvement.

### ISSUE THREAD

**Issue by benjaminapetersen** [operator-framework/operator-sdk#378]
For context, I'm following along with [this article about creating operators](https://devops.college/developing-kubernetes-operator-is-now-easy-with-operator-framework-d3194a7428ff).  After making a few changes I ran the `operator-sdk generate k8s` command, which generated this error:

<img width="482" alt="screen shot 2018-08-02 at 10 36 26 am 1" src="https://user-images.githubusercontent.com/280512/43590886-13b7d5f8-9640-11e8-826b-7e9187dc9350.png">

What is lacking is some direction for resolving the issue(s). I assume that there ought to be a list of CustomResources between the `( )` parens.  Ideally, there would also be imformation explaining why the code could not generate.  

Thanks!

**Comment by fanminshi** [comment]
@benjaminapetersen does this happen if you follow the [user_guide](https://github.com/operator-framework/operator-sdk/blob/master/doc/user-guide.md)?

**Comment by benjaminapetersen** [comment]
I was not following the user guide for this one, I was experimenting with the linked article & trying to move beyond the guide.

**Comment by fanminshi** [comment]
@benjaminapetersen yeah the error message needs to be better. However, Could you reproduce the error so that I can repeat it myself to determine what went wrong.

**Comment by benjaminapetersen** [comment]
Would you like me to share the files I created locally?

**Comment by fanminshi** [comment]
@benjaminapetersen it would be better if you can create a simple steps that can repeat this issue. If not, I can take a look of your files.

**Comment by grosser** [comment]
FYI ran into this on ubuntu machine and running the same on mac worked fine 🤷‍♂️ 
(using downloaded binaries from the release page)
